distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/TestFederation.java

```diff
@@ -197,28 +272,40 @@ public void afterAllianceTest() throws Exception {
             mockServerThread.interrupt();
         }
         mockServerThread = null;
+
+        if (cswServer != null) {
+            cswServer.stop();
+        }
     }
 
     private void startMockResources() throws Exception {
+        cswServer = new FederatedMgmpMockServer(CSW_STUB_SOURCE_ID, INSECURE_ROOT, Integer.parseInt(
```

Comment:
I see that the Nsili mock server is running in a new thread but it doesn't _appear_ on cursory examination that this server is?